CONTRIBUTING.md

```diff
@@ -53,20 +53,44 @@ Make sure to install **Ubuntu** as your Linux distribution. (This should be defa
 <details>
   <summary> Bank Users 🏦 </summary>
 
+  Pawnee Diaper Bank
   ```
     Organization Admin
-       Email: org_admin1@example.com
+    Email: org_admin1@example.com
     Password: password!
 
     User
     Email: user_1@example.com
     Password: password!
   ```
+
+  SF Diaper Bank
+  ```
+    Organization Admin
+    Email: org_admin2@example.com
+    Password: password!
+
+    User
+    Email: user_2@example.com
+    Password: password!
+  ```
+
+  Second City Essentials Bank
```

Comment:
Would it be worth it to specify what the difference is between these accounts?

Reply:
It certainly would.  

The biggest difference is that SF Diaper Bank is a brand new bank that hasn't been set up yet.   

Now that you mention it, I'd suggest putting ordering them as Pawnee, then Second City,   then SF,   with a note on SF Diaper bank at the end with a note that it is a freshly accepted bank (i.e. not yet set up)


We might tune Second City to have more particular characteristics as we find things that are hard to test on Pawnee,   but for now its main purpose is so we have multiple in-flight organizations while we test.